### PR TITLE
Add Instructions to README to Run on Real Device

### PR DESCRIPTION
**Pull request by cara-wong**
## Changes
Add instructions for running on a physical device without an official developer account.

## Requirements
N/A

## PR Dependencies
N/A

## Merged Branches and Other Dependencies
Merged branch(es): None
Other dependencies: None

## Subtasks / Progress
N/A

## Preview
N/A

## Testing
Manual.

### CODE REVIEW

**Comment by cara-wong** [comment]
@crugwiro @rlnsy Aidin wanted the instructions to run the project on a real device to be added to the readme. I added the mac instructions for running on an ios device. Can you guys verify these because it's been a while since I did this process from the beginning? Also have either of you ran the project on an android device and could add the instructions for that?

**Comment by rlnsy** [comment]
@cara-wong Thanks. I'll double check the ios process. I haven't ever tried running android. Can you verify it @crugwiro?

**Review by crugwiro — APPROVED** [review]
For Android, you need to enable USB debugging. Ref [https://reactnative.dev/docs/running-on-device](url)
1. enable "DEVELOPER OPTIONS" by going to Settings->About phone -> Software Information
2. Scroll to the bottom and tap on 'Build number' 7 times ( you will see a countdown to let you know when you get to 7)
3. Go back to Settings -> Developer options and enable 'USB Debugging'
4. Plug in your device via USB and run `adb devices` on your terminal (this helps you know if you're connected to the Android Debug Bridge)
5. you should see `List of devices attached` and make sure you see your device listed
6. you can run the app on your physical device by running `yarn android`

**Comment by crugwiro** [comment]
Also, these are *Mac instructions might be different on Windows.

**Review by rlnsy — APPROVED** [review]

**Comment by rlnsy** [comment]
@wynnset I tested this as well so if it works for you then it's ready to go.

**Comment by MarkDavies0** [comment]
Something minor I found when running on a physical iPhone:

When prompted, `"CompanionKit" would like to find and connect to local devices on your network` choose `OK` otherwise you will receive a bundle.js error and not be able to run. If you tapped `Don't Allow` delete CompanionKit from your iPhone and retry. 

It seems obvious but I tapped `Don't Allow` out of habit, might save someone else a few minutes.

File: mobile/ios/Podfile.lock [hunk 1]
```diff
@@ -191,6 +191,8 @@ PODS:
     - OpenSSL-Universal/Static (= 1.0.2.20)
   - OpenSSL-Universal/Static (1.0.2.20)
   - PromisesObjC (1.2.12)
+  - RCTAppleHealthKit (0.8.0):
```
Comment (wynnset) [inline mobile/ios/Podfile.lock:194]: @rlnsy Can you confirm this is meant to be part of this PR? I thought this PR is only for updating the readme?

Comment (rlnsy) [reply mobile/ios/Podfile.lock]: Removed

**Pull request merged**